ansible/roles/prometheus/templates/prometheus.yml.j2

```diff
@@ -51,7 +51,7 @@ scrape_configs:
       module: [{{ bbjob.module }}]
     static_configs:
       - targets:
-{% for target in bbjob.targets %}
+{% for target in (bbjob.targets|sort) %}
```

Comment:
Was the sort added to make the diff more deterministic? I am surprised that ansible did not do it automatically.

Reply:
Yes the first, no to the latter: the diff was random every time